test/db/esil/arm_32

```diff
@@ -589,6 +589,27 @@ r2 = 0xffeeddcc
 EOF
 RUN
 
+NAME=ldr r0, [pc, r0]
+FILE=malloc://0x200
+CMDS=<<EOF
+e asm.arch=arm
+e asm.bits=32
+s 0x20
+wx 00009fe7 # ldr r0, [pc, r0]
```

Comment:
could you also add a `pd 1` after this instruction?
I think this is another good test for what you are fixing in the other PR.

Reply:
added.
````
            0x00000020      ldr   r0, [pc, r0]                         ; [0x28:4]=2
````

The comment part
```` [0x28:4]=2 ````
should not be show , but this is another issue.